### ISSUE TITLE
Multi language support is unavailable

### ISSUE THREAD

**Issue by WinExp** [BlossomMods/BlossomLib#15]
I tried to use the command in the zh_cn language, but the command still returns English.
![2023-12-20_21 12 35](https://github.com/BlossomMods/BlossomLib/assets/55543397/28dec18f-78ac-4e7a-ac52-69794d7280d8)
![2023-12-20_21 12 55](https://github.com/BlossomMods/BlossomLib/assets/55543397/38a6102d-b5c5-4a60-b437-4a4fb6908f38)

**Comment by CodedSakura** [comment]
is this on single-player or multi-player?

**Comment by WinExp** [comment]
@CodedSakura On my multi-player server

**Comment by WinExp** [comment]
@CodedSakura And I can't find the configuration for the language option on server.

**Comment by WinExp** [comment]
@CodedSakura 在我进入服务器后，它输出了如下错误日志：
```
[12:33:04] [Netty Epoll Server IO #7/WARN]: Failed to load language resource at blossom:lang/zh_cn.json
com.google.gson.JsonSyntaxException: com.google.gson.stream.MalformedJsonException: Unterminated object at line 17 column 4 path $.blossom.homes.setMax.delimiter
	at com.google.gson.Gson.fromJson(Gson.java:1241) ~[gson-2.10.1.jar:?]
	at com.google.gson.Gson.fromJson(Gson.java:1137) ~[gson-2.10.1.jar:?]
	at com.google.gson.Gson.fromJson(Gson.java:1075) ~[gson-2.10.1.jar:?]
	at net.minecraft.class_2477.method_29425(class_2477.java:83) ~[server-intermediary.jar:?]
	at xyz.nucleoid.server.translations.impl.language.LanguageReader.read(LanguageReader.java:19) ~[server_translations_api-2.2.0+1.20.3-rc1-1a0d0a405b021681.jar:?]
	at xyz.nucleoid.server.translations.impl.language.LanguageReader.lambda$collectDataPackTranslations$1(LanguageReader.java:40) ~[server_translations_api-2.2.0+1.20.3-rc1-1a0d0a405b021681.jar:?]
	at xyz.nucleoid.server.translations.impl.language.TranslationStore.tryLoad(TranslationStore.java:50) ~[server_translations_api-2.2.0+1.20.3-rc1-1a0d0a405b021681.jar:?]
	at xyz.nucleoid.server.translations.impl.language.TranslationStore.get(TranslationStore.java:37) ~[server_translations_api-2.2.0+1.20.3-rc1-1a0d0a405b021681.jar:?]
	at xyz.nucleoid.server.translations.impl.ServerTranslations.createLanguage(ServerTranslations.java:122) ~[server_translations_api-2.2.0+1.20.3-rc1-1a0d0a405b021681.jar:?]
	at xyz.nucleoid.server.translations.impl.ServerTranslations.getLanguage(ServerTranslations.java:107) ~[server_translations_api-2.2.0+1.20.3-rc1-1a0d0a405b021681.jar:?]
	at xyz.nucleoid.server.translations.impl.ServerTranslations.getLanguage(ServerTranslations.java:91) ~[server_translations_api-2.2.0+1.20.3-rc1-1a0d0a405b021681.jar:?]
	at xyz.nucleoid.server.translations.api.LocalizationTarget.getLanguage(LocalizationTarget.java:28) ~[server_translations_api-2.2.0+1.20.3-rc1-1a0d0a405b021681.jar:?]
	at net.minecraft.class_2588.md570d23$server_translations_api$lambda$stapi$addTranslationFallback$0$0(class_2588.java:1529) ~[server-intermediary.jar:?]
	at com.mojang.serialization.MapEncoder$1.encode(MapEncoder.java:26) ~[datafixerupper-6.0.8.jar:?]
	at com.mojang.serialization.MapCodec$1.encode(MapCodec.java:39) ~[datafixerupper-6.0.8.jar:?]
	at net.minecraft.class_8824$class_8825.encode(class_8824.java:101) ~[server-intermediary.jar:?]
	at net.minecraft.class_8824$class_8912.encode(class_8824.java:67) ~[server-intermediary.jar:?]
	at net.minecraft.class_5699$3.encode(class_5699.java:246) ~[server-intermediary.jar:?]
	at com.mojang.serialization.codecs.RecordCodecBuilder$Instance$6.encode(RecordCodecBuilder.java:295) ~[datafixerupper-6.0.8.jar:?]
	at com.mojang.serialization.codecs.RecordCodecBuilder$2.encode(RecordCodecBuilder.java:112) ~[datafixerupper-6.0.8.jar:?]
	at com.mojang.serialization.MapCodec$MapCodecCodec.encode(MapCodec.java:96) ~[datafixerupper-6.0.8.jar:?]
	at com.mojang.serialization.codecs.EitherCodec.lambda$encode$3(EitherCodec.java:35) ~[datafixerupper-6.0.8.jar:?]
	at com.mojang.datafixers.util.Either$Right.map(Either.java:99) ~[datafixerupper-6.0.8.jar:?]
	at com.mojang.serialization.codecs.EitherCodec.encode(EitherCodec.java:33) ~[datafixerupper-6.0.8.jar:?]
	at com.mojang.serialization.codecs.EitherCodec.encode(EitherCodec.java:13) ~[datafixerupper-6.0.8.jar:?]
	at com.mojang.serialization.Encoder$1.encode(Encoder.java:25) ~[datafixerupper-6.0.8.jar:?]
	at com.mojang.serialization.Codec$2.encode(Codec.java:76) ~[datafixerupper-6.0.8.jar:?]
	at net.minecraft.class_5699$class_8741.encode(class_5699.java:517) ~[server-intermediary.jar:?]
	at com.mojang.serialization.Encoder.encodeStart(Encoder.java:14) ~[datafixerupper-6.0.8.jar:?]
	at com.mojang.serialization.codecs.FieldEncoder.encode(FieldEncoder.java:24) ~[datafixerupper-6.0.8.jar:?]
	at com.mojang.serialization.MapCodec$1.encode(MapCodec.java:39) ~[datafixerupper-6.0.8.jar:?]
	at com.mojang.serialization.codecs.RecordCodecBuilder$Instance$6.encode(RecordCodecBuilder.java:297) ~[datafixerupper-6.0.8.jar:?]
	at com.mojang.serialization.codecs.RecordCodecBuilder$2.encode(RecordCodecBuilder.java:112) ~[datafixerupper-6.0.8.jar:?]
	at com.mojang.serialization.MapCodec$MapCodecCodec.encode(MapCodec.java:96) ~[datafixerupper-6.0.8.jar:?]
	at com.mojang.serialization.Encoder.encodeStart(Encoder.java:14) ~[datafixerupper-6.0.8.jar:?]
	at com.mojang.serialization.codecs.FieldEncoder.encode(FieldEncoder.java:24) ~[datafixerupper-6.0.8.jar:?]
	at com.mojang.serialization.MapCodec$1.encode(MapCodec.java:39) ~[datafixerupper-6.0.8.jar:?]
	at com.mojang.serialization.codecs.RecordCodecBuilder$Instance$6.encode(RecordCodecBuilder.java:297) ~[datafixerupper-6.0.8.jar:?]
	at com.mojang.serialization.codecs.RecordCodecBuilder$2.encode(RecordCodecBuilder.java:112) ~[datafixerupper-6.0.8.jar:?]
	at com.mojang.serialization.MapCodec$MapCodecCodec.encode(MapCodec.java:96) ~[datafixerupper-6.0.8.jar:?]
	at com.mojang.serialization.Encoder.encodeStart(Encoder.java:14) ~[datafixerupper-6.0.8.jar:?]
	at com.mojang.serialization.codecs.ListCodec.encode(ListCodec.java:31) ~[datafixerupper-6.0.8.jar:?]
	at com.mojang.serialization.codecs.ListCodec.encode(ListCodec.java:19) ~[datafixerupper-6.0.8.jar:?]
	at com.mojang.serialization.Encoder$1.encode(Encoder.java:25) ~[datafixerupper-6.0.8.jar:?]
	at com.mojang.serialization.Codec$2.encode(Codec.java:76) ~[datafixerupper-6.0.8.jar:?]
	at com.mojang.serialization.Encoder.encodeStart(Encoder.java:14) ~[datafixerupper-6.0.8.jar:?]
	at com.mojang.serialization.codecs.KeyDispatchCodec.encode(KeyDispatchCodec.java:98) ~[datafixerupper-6.0.8.jar:?]
	at com.mojang.serialization.MapCodec$MapCodecCodec.encode(MapCodec.java:96) ~[datafixerupper-6.0.8.jar:?]
	at com.mojang.serialization.Encoder.encodeStart(Encoder.java:14) ~[datafixerupper-6.0.8.jar:?]
	at com.mojang.serialization.codecs.BaseMapCodec.encode(BaseMapCodec.java:52) ~[datafixerupper-6.0.8.jar:?]
	at com.mojang.serialization.codecs.UnboundedMapCodec.encode(UnboundedMapCodec.java:43) ~[datafixerupper-6.0.8.jar:?]
	at com.mojang.serialization.codecs.UnboundedMapCodec.encode(UnboundedMapCodec.java:17) ~[datafixerupper-6.0.8.jar:?]
	at com.mojang.serialization.Encoder$1.encode(Encoder.java:25) ~[datafixerupper-6.0.8.jar:?]
	at com.mojang.serialization.Codec$2.encode(Codec.java:76) ~[datafixerupper-6.0.8.jar:?]
	at com.mojang.serialization.Encoder.encodeStart(Encoder.java:14) ~[datafixerupper-6.0.8.jar:?]
	at net.minecraft.class_2540.method_29172(class_2540.java:135) ~[server-intermediary.jar:?]
	at net.minecraft.class_8734.method_11052(class_8734.java:26) ~[server-intermediary.jar:?]
	at net.minecraft.class_2545.method_10838(class_2545.java:47) ~[server-intermediary.jar:?]
	at net.minecraft.class_2545.encode(class_2545.java:15) ~[server-intermediary.jar:?]
	at io.netty.handler.codec.MessageToByteEncoder.write(MessageToByteEncoder.java:107) ~[netty-codec-4.1.97.Final.jar:?]
	at io.netty.channel.AbstractChannelHandlerContext.invokeWrite0(AbstractChannelHandlerContext.java:881) ~[netty-transport-4.1.97.Final.jar:?]
	at io.netty.channel.AbstractChannelHandlerContext.invokeWrite(AbstractChannelHandlerContext.java:863) ~[netty-transport-4.1.97.Final.jar:?]
	at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:968) ~[netty-transport-4.1.97.Final.jar:?]
	at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:856) ~[netty-transport-4.1.97.Final.jar:?]
	at io.netty.handler.codec.MessageToMessageEncoder.write(MessageToMessageEncoder.java:113) ~[netty-codec-4.1.97.Final.jar:?]
	at io.netty.channel.AbstractChannelHandlerContext.invokeWrite0(AbstractChannelHandlerContext.java:881) ~[netty-transport-4.1.97.Final.jar:?]
	at io.netty.channel.AbstractChannelHandlerContext.invokeWriteAndFlush(AbstractChannelHandlerContext.java:940) ~[netty-transport-4.1.97.Final.jar:?]
	at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:966) ~[netty-transport-4.1.97.Final.jar:?]
	at io.netty.channel.AbstractChannelHandlerContext.writeAndFlush(AbstractChannelHandlerContext.java:934) ~[netty-transport-4.1.97.Final.jar:?]
	at io.netty.channel.AbstractChannelHandlerContext.writeAndFlush(AbstractChannelHandlerContext.java:984) ~[netty-transport-4.1.97.Final.jar:?]
	at io.netty.channel.DefaultChannelPipeline.writeAndFlush(DefaultChannelPipeline.java:1025) ~[netty-transport-4.1.97.Final.jar:?]
	at io.netty.channel.AbstractChannel.writeAndFlush(AbstractChannel.java:306) ~[netty-transport-4.1.97.Final.jar:?]
	at net.minecraft.class_2535.method_36942(class_2535.java:321) ~[server-intermediary.jar:?]
	at net.minecraft.class_2535.method_10764(class_2535.java:314) ~[server-intermediary.jar:?]
	at net.minecraft.class_2535.method_52906(class_2535.java:296) ~[server-intermediary.jar:?]
	at net.minecraft.class_8609.method_52391(class_8609.java:119) ~[server-intermediary.jar:?]
	at net.minecraft.class_8609.method_14364(class_8609.java:113) ~[server-intermediary.jar:?]
	at net.minecraft.class_8610.method_52409(class_8610.java:73) ~[server-intermediary.jar:?]
	at net.minecraft.class_8610.completeTask(class_8610.java:673) ~[server-intermediary.jar:?]
	at net.fabricmc.fabric.impl.registry.sync.FabricRegistryInit.lambda$onInitialize$0(FabricRegistryInit.java:35) ~[fabric-registry-sync-v0-4.0.13+78d798af4f-ee63f30ec0934686.jar:?]
	at net.fabricmc.fabric.api.networking.v1.ServerConfigurationNetworking.lambda$wrapUntyped$0(ServerConfigurationNetworking.java:355) ~[fabric-networking-api-v1-3.1.2+78d798af4f-64c1ea16245c0220.jar:?]
	at net.fabricmc.fabric.impl.networking.server.ServerConfigurationNetworkAddon.receive(ServerConfigurationNetworkAddon.java:107) ~[fabric-networking-api-v1-3.1.2+78d798af4f-64c1ea16245c0220.jar:?]
	at net.fabricmc.fabric.impl.networking.server.ServerConfigurationNetworkAddon.receive(ServerConfigurationNetworkAddon.java:43) ~[fabric-networking-api-v1-3.1.2+78d798af4f-64c1ea16245c0220.jar:?]
	at net.fabricmc.fabric.impl.networking.AbstractChanneledNetworkAddon.handle(AbstractChanneledNetworkAddon.java:99) ~[fabric-networking-api-v1-3.1.2+78d798af4f-64c1ea16245c0220.jar:?]
	at net.minecraft.class_8609.handler$ckf000$fabric-networking-api-v1$handleCustomPayloadReceivedAsync(class_8609.java:2044) ~[server-intermediary.jar:?]
	at net.minecraft.class_8609.method_52392(class_8609.java) ~[server-intermediary.jar:?]
	at net.minecraft.class_2817.method_12199(class_2817.java:54) ~[server-intermediary.jar:?]
	at net.minecraft.class_2817.method_11054(class_2817.java:13) ~[server-intermediary.jar:?]
	at net.minecraft.class_2535.method_10759(class_2535.java:196) ~[server-intermediary.jar:?]
	at net.minecraft.class_2535.method_10770(class_2535.java:181) ~[server-intermediary.jar:?]
	at net.minecraft.class_2535.channelRead0(class_2535.java:60) ~[server-intermediary.jar:?]
	at io.netty.channel.SimpleChannelInboundHandler.channelRead(SimpleChannelInboundHandler.java:99) ~[netty-transport-4.1.97.Final.jar:?]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:444) ~[netty-transport-4.1.97.Final.jar:?]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420) ~[netty-transport-4.1.97.Final.jar:?]
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:412) ~[netty-transport-4.1.97.Final.jar:?]
	at io.netty.handler.flow.FlowControlHandler.dequeue(FlowControlHandler.java:202) ~[netty-handler-4.1.97.Final.jar:?]
	at io.netty.handler.flow.FlowControlHandler.channelRead(FlowControlHandler.java:164) ~[netty-handler-4.1.97.Final.jar:?]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:442) ~[netty-transport-4.1.97.Final.jar:?]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420) ~[netty-transport-4.1.97.Final.jar:?]
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:412) ~[netty-transport-4.1.97.Final.jar:?]
	at io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:103) ~[netty-codec-4.1.97.Final.jar:?]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:444) ~[netty-transport-4.1.97.Final.jar:?]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420) ~[netty-transport-4.1.97.Final.jar:?]
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:412) ~[netty-transport-4.1.97.Final.jar:?]
	at io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:346) ~[netty-codec-4.1.97.Final.jar:?]
	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:318) ~[netty-codec-4.1.97.Final.jar:?]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:444) ~[netty-transport-4.1.97.Final.jar:?]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420) ~[netty-transport-4.1.97.Final.jar:?]
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:412) ~[netty-transport-4.1.97.Final.jar:?]
	at io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:346) ~[netty-codec-4.1.97.Final.jar:?]
	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:318) ~[netty-codec-4.1.97.Final.jar:?]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:444) ~[netty-transport-4.1.97.Final.jar:?]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420) ~[netty-transport-4.1.97.Final.jar:?]
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:412) ~[netty-transport-4.1.97.Final.jar:?]
	at io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:346) ~[netty-codec-4.1.97.Final.jar:?]
	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:318) ~[netty-codec-4.1.97.Final.jar:?]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:444) ~[netty-transport-4.1.97.Final.jar:?]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420) ~[netty-transport-4.1.97.Final.jar:?]
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:412) ~[netty-transport-4.1.97.Final.jar:?]
	at io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:103) ~[netty-codec-4.1.97.Final.jar:?]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:444) ~[netty-transport-4.1.97.Final.jar:?]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420) ~[netty-transport-4.1.97.Final.jar:?]
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:412) ~[netty-transport-4.1.97.Final.jar:?]
	at io.netty.handler.timeout.IdleStateHandler.channelRead(IdleStateHandler.java:286) ~[netty-handler-4.1.97.Final.jar:?]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:442) ~[netty-transport-4.1.97.Final.jar:?]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420) ~[netty-transport-4.1.97.Final.jar:?]
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:412) ~[netty-transport-4.1.97.Final.jar:?]
	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1410) ~[netty-transport-4.1.97.Final.jar:?]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:440) ~[netty-transport-4.1.97.Final.jar:?]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420) ~[netty-transport-4.1.97.Final.jar:?]
	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:919) ~[netty-transport-4.1.97.Final.jar:?]
	at io.netty.channel.epoll.AbstractEpollStreamChannel$EpollStreamUnsafe.epollInReady(AbstractEpollStreamChannel.java:800) ~[netty-transport-classes-epoll-4.1.97.Final.jar:?]
	at io.netty.channel.epoll.EpollEventLoop.processReady(EpollEventLoop.java:509) ~[netty-transport-classes-epoll-4.1.97.Final.jar:?]
	at io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:407) ~[netty-transport-classes-epoll-4.1.97.Final.jar:?]
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997) ~[netty-common-4.1.97.Final.jar:?]
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) ~[netty-common-4.1.97.Final.jar:?]
	at java.lang.Thread.run(Thread.java:1583) ~[?:?]
Caused by: com.google.gson.stream.MalformedJsonException: Unterminated object at line 17 column 4 path $.blossom.homes.setMax.delimiter
	at com.google.gson.stream.JsonReader.syntaxError(JsonReader.java:1659) ~[gson-2.10.1.jar:?]
	at com.google.gson.stream.JsonReader.doPeek(JsonReader.java:500) ~[gson-2.10.1.jar:?]
	at com.google.gson.stream.JsonReader.hasNext(JsonReader.java:422) ~[gson-2.10.1.jar:?]
	at com.google.gson.internal.bind.TypeAdapters$28.read(TypeAdapters.java:779) ~[gson-2.10.1.jar:?]
	at com.google.gson.internal.bind.TypeAdapters$28.read(TypeAdapters.java:725) ~[gson-2.10.1.jar:?]
	at com.google.gson.internal.bind.TypeAdapters$34$1.read(TypeAdapters.java:1007) ~[gson-2.10.1.jar:?]
	at com.google.gson.Gson.fromJson(Gson.java:1227) ~[gson-2.10.1.jar:?]
	... 136 more
```

**Comment by CodedSakura** [comment]
oh, right, there's a missing comma in the zh_cn translation file. you can fix it by using a data pack with a corrected translation file (afaik), but i should probably fix this in the releases

which version of minecraft are you using?

**Comment by WinExp** [comment]
@CodedSakura 1.20.4

**Comment by WinExp** [comment]
@CodedSakura Sorry I replied in Chinese :)

**Comment by WinExp** [comment]
I create a pull request to fix it

**Comment by CodedSakura** [comment]
thanks! i'll create a release soon

one thing tho, the english translation file has that delimiter as `","`, is it correct that there should be no comma separating player names?

**Comment by WinExp** [comment]
Oh sorry i forgot the comma，sry about that：）

**Comment by WinExp** [comment]
I think in both Chinese and English there must be a comma separating player names, my mistake: )

**Comment by CodedSakura** [comment]
fixed in the latest (1.20.6) version of blossom homes (2.2.3)
